com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/mqtt_streams.py

```diff
@@ -0,0 +1,208 @@
+
+from streamsx.topology import graph
+from streamsx.topology.topology import *
+from streamsx.topology import schema
+from enum import Enum
+
+
+class mqtt_streams(object):
```

Comment:
Class names should normally use the CapWords convention.

https://www.python.org/dev/peps/pep-0008/#class-names